views/default/panels/db/detail.php

```diff
@@ -3,13 +3,20 @@
 /* @var $searchModel yii\debug\models\search\Db */
 /* @var $dataProvider yii\data\ArrayDataProvider */
 /* @var $hasExplain bool */
+/* @var $sumDuplicates int */
 
-use yii\helpers\Html;
 use yii\grid\GridView;
+use yii\helpers\Html;
 use yii\web\View;
 
 echo Html::tag('h1', $panel->getName() . ' Queries');
 
+if ($sumDuplicates === 1) {
```

Comment:
Why not `Yii::t()`?

Reply:
No need to translate here. Formatter would be fine.

Reply:
`echo Yii::t('app', '{duplicates,plural,=0{No} other{#}} duplicated {duplicates,plural,=0{query} =1{query} other{queries}} found.', ['duplicates' =>$sumDuplicates]);`
Output: 
"No duplicated query found." when $sumDuplicates = 0;
"1 duplicated query found." when $sumDuplicates = 1;
"2 duplicated queries found." when $sumDuplicates > 2;

It's ok? (need check before commit :))

Reply:
@SilverFire we're not translating debug toolbar i.e. there's no `Yii::t()` in there at all.

Reply:
Ok

Reply:
I think current code is fine.